partner_communication_switzerland/models/correspondence.py

```diff
@@ -225,7 +225,7 @@ def send_unread_b2s(self):
         """
         unread_config = self.env.ref(
             'partner_communication_switzerland.child_letter_unread')
-        for letter in self:
+        for letter in self.filtered(lambda c: c.letter_delivered):
```

Comment:
Changes are good, except it's the opposite! We want to send the reminder to those for which the letter is not delivered. Please fix conflicts as well.
```suggestion
        for letter in self.filtered(lambda c: not c.letter_delivered):
```